src/exodus/wallet.h

```diff
@@ -0,0 +1,71 @@
+#ifndef EXODUS_WALLET_H
```

Comment:
We should prefix with `ZCOIN` to prevent conflicting with installed header files.